src/lib/one-graph/netlify-graph.js

```diff
@@ -0,0 +1,705 @@
+const { randomUUID } = require('crypto')
+const fs = require('fs')
+
+const dotProp = require('dot-prop')
+const { parse, print, printSchema } = require('graphql')
+
+const { getFunctionsDir } = require('../../utils')
+
+const {
+  patchSubscriptionWebhookField,
+  patchSubscriptionWebhookSecretField,
+  typeScriptSignatureForOperation,
+  typeScriptSignatureForOperationVariables,
+} = require('./graphql-helpers')
+const { computeOperationDataList, netlifyFunctionSnippet } = require('./netlify-graph-code-exporter-snippets')
+
+const capitalizeFirstLetter = (string) => string.charAt(0).toUpperCase() + string.slice(1)
+
+const replaceAll = (target, search, replace) => {
+  const simpleString = search.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')
+  return target.replace(new RegExp(simpleString, 'g'), replace)
+}
+
+const sourceOperationsFilename = 'netligraphOperationsLibrary.graphql'
+
+const defaultNetligraphConfig = {
+  extension: 'mjs',
+  netligraphPath: 'netlify',
+  moduleType: 'commonjs',
```

Comment:
This does not play well in my head .mjs stands for module format and not commonjs? 

Maybe this is a typo